### DISCUSSION TITLE
How can I publish the  living camera streaming from Embedded development board  to  server by using gstreamer ?

### DISCUSSION THREAD

**Discussion by justtry94** [bluenviron/mediamtx#280] [Questions]
## Which version are you using?

rtsp-simple-server_v0.13.2_windows_amd64

## Question
Is it possible to publish the  live camera streaming from embedded development board  to  rtsp server on host(WIN10)  by using gstreamer gst-launch command ?   I can't find so much Plugins which are related with rtsp . For example no element  named rtspsink .
Description

**Comment by aler9** [comment]
Hello, of course it's possible to publish from Gstreamer, as written in the README, the right element is `rtspclientsink`

```
gst-launch-1.0 rtspclientsink name=s location=rtsp://localhost:8554/mystream filesrc location=file.mp4 ! qtdemux name=d d.video_0 ! queue ! s.sink_0 d.audio_0 ! queue ! s.sink_1
```

You didn't mention the name and OS of the board, hence i have no way to help you finding out the problem.

[a comment by an automated account is not shown]
